.github/workflows/main.yml

```diff
@@ -13,6 +13,8 @@ on:
       - '**/**.js.snap'
       - '.github/workflows/*'
   push:
+    tags:
+      - 'v*'
```

Comment:
@tabiodun ahh! this snuck back in here!

Reply:
Just removed it lol

Reply:
haha, for real though this PR is awesome :100:

Reply:
👍